### DISCUSSION TITLE
Becoming a Microsoft MVP

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#59] [General]
# Becoming a Microsoft MVP

In this article, I will talk about my journey to becoming a Microsoft MVP. I will talk about the process, the requirements, and the benefits of becoming a Microsoft MVP. Of course, this is my personal experience and journey, so mileage may vary.

https://steven-giesel.com/blogPost/2809a9a9-66e3-4928-a5fb-6abbdbfdf272

**Comment by dimohy** [comment]
Congratulations!

I really enjoy your blog and your writing. I look forward to reading your blog for years to come.

**Reply by linkdotnet** [reply]
Hey, thanks for your supportive and nice words! That really means a lot to me. I hope I can continue the journey and try out different things here and there!
